app/views/nutriologo_pages/mis_sesiones.html.erb

```diff
@@ -17,8 +17,7 @@
         <td><%= getPacienteName(sesion.paciente_id) %></td>
         <td><%= getSede(sesion.hora_id) %></td>
         <td><%= getHora(sesion.hora_id) %></td>
-
-        <!-- editar sesión actual<td><%= link_to 'Editar datos', edit_sesion_path(sesion), class: "btn-small pink lighten-2 btn-block" %></td>-->
+        
```

Comment:
Exacto, no dejamos codigo commentado, se vuelve sucio, aqui lo que puedes hacer es un `git rebase -i HEAD~2` en tu terminal te apareceran varias lineas en el ultimo commit de `eliminar boton datos ` cambia la parte que dice `pick` por un `squash` esto lo que hace es te junta ese commit con el anterior, de tal manera ya solo te queda un solo commit. Esto es bueno porque mantenemos un historial de commit limpio y evita esta tipo de confusiones :) Igual cualqueir duda no dudes en preguntarme! Te dejo un link por si quieres leer mas sobre github esta interesante 

https://medium.com/@cvortmann/what-makes-a-good-commit-message-995d23687ad
https://dev.to/pavlosisaris/git-commits-an-effective-style-guide-2kkn
https://chris.beams.io/posts/git-commit/

[REBASE](https://thoughtbot.com/blog/git-interactive-rebase-squash-amend-rewriting-history)

Reply:
Gracias la información muy enriquecedora y complementaria para mis proximas modificaciones.